octree/include/pcl/octree/octree_base.h

```diff
@@ -247,6 +247,10 @@ class OctreeBase {
   {
     leaf_count_ = source.leaf_count_;
     branch_count_ = source.branch_count_;
+    if (root_node_)
+    {
+        delete root_node_;
+    }
```

Comment:
```suggestion
    if (root_node_) {
      delete root_node_;
    }
```

Reply:
The null pointer check is not required.

> In all cases, if ptr is a null pointer, the standard library deallocation functions do nothing. 

[Source](https://en.cppreference.com/w/cpp/memory/new/operator_delete)

Reply:
😢

Reply:
Oops, sorry for merging too fast! :grimacing: